src/components/can_cooperation/test/include/mock_application_some_impl.h

```diff
@@ -0,0 +1,332 @@
+/*
```

Comment:
If we move SOToInt into generic module and create method which will be return vector<int> for hmi types we can cut up this dependence from SO. I hope...